### ISSUE TITLE
Items in Masonry view should receive keyboard focus

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/exhibits#2465]
### Current behavior

When navigating by keyboard Tab/Shift+Tab (or by VoiceOver screen reader), the focus skips over the image items in Masonry view.

### Steps to reproduce
Example page: [Parker Library - Search results - Masonry view](https://exhibits.stanford.edu/parker/catalog?f%5Bformat_main_ssim%5D%5B%5D=Archive%2FManuscript&search_field=manuscript_number&sort=title_sort+asc%2C+pub_year_isi+desc&view=masonry).

Use Tab/Shift+Tab and Enter keys to navigate through the Search result items in masonry view.

### Possible solution

Note that when hovering over the image with the mouse the item's metadata appears as a clickable link. A possible solution may be to display and focus on this link when navigating by keyboard.

### Reported by SODA
> [Issue]
> There are actionable elements that are not functional with the keyboard. The masonry view lacks keyboard support for each element and it is not possible to activate any element to view details from this view. 
> 
> [User Impact]
> All functionality must be actionable regardless of the input method used. Navigation and functionality that is dependent on specific devices such as a mouse may not be available to users with dexterity impairments as well as people without vision or low vision. These users may not be able to use a mouse so content providers should ensure that all functionality can be used from the keyboard.
> 
> [Code Reference]
> <a data-context-href=""/catalog/vb856kp8798/track?counter=4&amp;document_id=vb856kp8798&amp;search_id=209806423"" aria-hidden=""true"" tabindex=""-1"" href=""/parker/catalog/vb856kp8798""><img alt="""" class=""img-thumbnail"" src=""https://stacks.stanford.edu/image/iiif/vb856kp8798%2F002II_122_R_NC_46/full/!400,400/0/default.jpg""></a>
> 
> [Recommendation]
> Developers should ensure that Keyboard access is provided to focus the element as well as keyboard access to interact with and activate the element. 
> 
> There is an aria-hidden and negative tabindex on the anhcor that needs to be removed. In addition, the image needs to have an ALT text.
>

**Comment by corylown** [comment]
Fixed by https://github.com/sul-dlss/SearchWorks/pull/4520/files released in blacklight-gallery v4.6.3.
